husky_bringup/launch/realsense_config/realsense.launch

```diff
@@ -0,0 +1,142 @@
+<launch>
+  <arg name="serial_no"           default=""/>
+  <arg name="usb_port_id"         default=""/>
+  <arg name="device_type"         default=""/>
+  <arg name="json_file_path"      default=""/>
+  <arg name="camera"              default="camera"/>
+  <arg name="tf_prefix"           default="$(arg camera)"/>
+  <arg name="external_manager"    default="false"/>
+  <arg name="manager"             default="realsense2_camera_manager"/>
+  <arg name="base_frame_id"       default="sensor_arch_mount_link" />
```

Comment:
You might want to make this an envar.

Reply:
Added a couple more environment variables to override these.